### PR TITLE
add lang prop to radio schedule section

### LINKED ISSUE #6869: Bugs - Persian/Afghanistan IDX page - Radio Schedule for Dari service
**Context**

Farsi is the main language for the Persian service. `lang="fa"`. Dari is a different dialect `lang="fa-AF"` - it is a regional dialect of Farsi spoken in Afghanistan.

**Describe the bug**

As part of our accessibility swarm of the `/persian/afghanistan` index, we found the following issues:

As a user to the website, there is no way I can tell that the radio schedule or radio programmes are in Dari and not in Farsi. Not via the lang attribute for screenreaders or via any indication on the page itself (title text on the radio schedule, or text on the on demand radio pages).
The titles/summaries on the Radio Schedule programmes are indistinguishable between Farsi and Dari. It’s only when you listen to the audio that you realise it’s in Dari.

- The radio schedule component is in the Dari dialect of Persian, but nothing on the page indicates that the content is in Dari. Instead of the h2 saying ‘Radio programmes’ (`برنامه‌های رادیو`) it should be ‘Radio programmes in Dari’. I don't believe the actual programme titles are written in the Dari dialect, otherwise they'd need to have `lang="fa-AF"`. Will need to discuss this with the Media team, since they are building the pages playing the content. 

- On this IDX page (`https://www.stage.bbc.com/persian/afghanistan?renderer_env=live`), when you scroll to the radio schedule (which has Dari radio programmes) and click through to an on-demand radio page, you can’t tell that the content is in Dari, since the logo just says BBC Farsi. Only part of the URL has '_dari_', but most people don’t check the url path. The <main> content of this page should have `lang="fa-AF"` e.g. example: `https://www.stage.bbc.com/persian/bbc_dari_radio/w172xfjyg5ph0j3`

- Across all services, in the radio schedule duration, we have the format 03:00 for a duration of 3 minutes where it should be 3:00 with no leading 0.  (The numerals themselves are correctly localised for this page.)

- For the radio schedule, there are multiple different programmes with the same title, just different start times in the timeline section. It’s not very clear to someone looking at it that these are 4 different episodes. It looks like it's just the same one repeated. Dat source: `https://www.bbc.com/persian/bbc_dari_radio/schedule.json`
Discuss the best solution with UX/Product. In terms of accessibility, the time is read out just prior to the date by screenreaders, so it is clear for screenreader users.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to `https://www.stage.bbc.com/persian/afghanistan?renderer_env=live`
2. Scroll down to Radio schedule - currently titled `برنامه‌های رادیو`
3. See points raised above.

**Expected behaviour**

**Screenshots**
Radio schedule for the Dari service on the persian/afghanistan page:
Screenshot persian afghanistan dari radio schedule

**Desktop (please complete the following information):**

- OS: macOS
- Browser Chrome
- Version 83

The issues raised above are related to all browsers, not just that listed above. 

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by thekp**
Resolves #6869

**Overall change:**
_Pass in lang prop from IDX page down into radio schedule container section wrapper._

**Code changes:**

- _Add new props to radio schedule container._
- _Hard code lang prop in IDX page_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/RadioSchedule/Canonical/index.test.jsx [hunk 1]
```diff
@@ -108,6 +112,20 @@ describe('Canonical RadioSchedule', () => {
       });
     });
 
+    it('render radio schedules container with lang code', async () => {
+      fetchMock.mock(endpoint, arabicRadioScheduleData);
+      let container;
+
+      await act(async () => {
+        container = await render(<RadioScheduleWithContext lang="khoa" />)
```
Comment (greenc05) [inline src/app/containers/RadioSchedule/Canonical/index.test.jsx]: khoa lang?!

Comment (thekp) [reply src/app/containers/RadioSchedule/Canonical/index.test.jsx]: 🙈

**Review by sareh — APPROVED** [review]
Looks good. When I load the page locally, I see the lang attribute added correctly on the radio schedule.

http://localhost:7080/persian/afghanistan
Screenshot lang attribute on radio schedule

**Review by tochwill — APPROVED** [review]

**Comment by PriyaKR** [comment]
Looks good to me on test env.

**Pull request merged**